lib/vanagon/driver.rb

```diff
@@ -164,6 +182,16 @@ def run # rubocop:disable Metrics/AbcSize, Metrics/PerceivedComplexity
       end
     end
 
+    def run
+      if @build_with_docker
+        warn "Building the project with docker ..."
```

Comment:
info?

Reply:
Unfortunately there is no `info` method defined.

Reply:
`puts`?